src/spyglass/lfp/v1/__init__.py

```diff
@@ -10,3 +10,6 @@
     LFPArtifactDetectionSelection,
     LFPArtifactRemovedIntervalList,
 )
+
+# Commented out to prevent circular import
+# from spyglass.lfp.v1.lfp_band import LFPBandV1, LFPBandSelection
```

Comment:
Would be good to figure out a way to include these in the namespace without causing a circular import.